e2e/basic/bittwister_test.go

```diff
@@ -72,41 +72,17 @@ func TestBittwister_Bandwidth(t *testing.T) {
 		name             string
 		targetBandwidth  int64
 		tolerancePercent int
-	}{{
-		name:             "512 Kbps",
-		targetBandwidth:  512 * 1000,
-		tolerancePercent: 50,
-	},
-		{
-			name:             "1 Mbps",
-			targetBandwidth:  1024 * 1000,
-			tolerancePercent: 50,
-		},
+	}{
 		{
-			name:             "2 Mbps",
-			targetBandwidth:  2 * 1024 * 1000,
+			name:             "512 Kbps",
+			targetBandwidth:  512 * 1000,
 			tolerancePercent: 50,
```

Comment:
50% deviation is pretty high isn't it? 
if this is a testing limitation then we should have a comment explaining the limitations on testing. 
Otherwise I would think we should have a much lower tolerance, like 10% or something. 

Also you could remove the table test structure now since you just have the one test.

Reply:
or you could leave in the table test and have 1 hard coded test and 1 random test.

Reply:
Well 50% is pretty high, true. Locally we do not need that high tolerance, but when test on cluster, or run a large test, sometimes the tolerance goes higher . Probably it is due to the fact that the time is a factor there.

There are two tests, so it is cleaner to keep the table.